test/translate.ts

```diff
@@ -1,8 +1,9 @@
-const fs = require('fs');
-const path = require('path');
-const tape = require('tape');
-const translate = require('../translate.js');
-const versionToSemver = translate.versionToSemver;
+import * as fs from 'fs'
```

Comment:
This file is inconsystent in not using semicolons for terminating.

Reply:
Initially the PR included changes that made semicolons consistent. I suggested that changes like these could be in the style refactor PR but they were removed. I still think this could be done, just as a separate PR. While reviewing here I mostly ignored that for that reason.